sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -31,64 +31,38 @@
 
 import operator
 import pandas
+import numpy
+
+import sdc
 
 from numba import types
 from numba.extending import (overload, overload_method, overload_attribute)
+from sdc.hiframes.pd_dataframe_ext import DataFrameType
 from numba.errors import TypingError
-
-from sdc.datatypes.hpat_pandas_dataframe_types import DataFrameType
-
-
-@overload_method(DataFrameType, 'count')
```

Comment:
Why did you remove the overload?

Reply:
Looks like this can be done in separate PR.

Reply:
This overload_method `count` is not used. It is ok to remove it. It looks like implementation of `sdc_pandas_dataframe_reduce_columns` function is preparation for other PR #363 
not sure in it.